src/onnx/parse_slice.cpp

```diff
@@ -77,7 +78,37 @@ struct parse_slice : op_parser<parse_slice>
             op.axes = axes;
         }
 
-        return info.add_instruction(op, args[0]);
+        std::vector<int64_t> raxes;
+        if(std::any_of(steps.begin(), steps.end(), [](auto s) { return s < 0; }))
+        {
+            for(auto i : op.axes)
+            {
+                if(steps[i] < 0)
+                    raxes.push_back(op.axes[i]); // populate negative only axis indices
+            }
+            
+            auto lens = args[0]->get_shape().lens();
```

Comment:
It looks like this variable is never used.

Reply:
Artifact, removed.